src/styles/ThemeStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+export const theme = createMuiTheme({
+    palette: {
+        primary: {
+        main: "#E5E5E5",
```

Comment:
update these colors to match Ashley's final color decisions once she's finished :) Reach out to her ab these!